demos/xss-attack-prevention/nested/microsoft.loganalytics/workspaces.json

```diff
@@ -99,7 +101,7 @@
       },
       "workspaceKey": {
         "type": "string",
-        "value": "[listKeys(resourceId('Microsoft.OperationalInsights/workspaces/', parameters('omsWorkspaceName')), '2017-04-26-preview').primarySharedKey]"
+        "value": "[listKeys(resourceId('Microsoft.OperationalInsights/workspaces/', parameters('omsWorkspaceName')), '2020-08-01').primarySharedKey]"
```

Comment:
this needs to be remove from the outputs (similar to the connstr changes we made).  You can output the resourceId and put the listKeys in the template that needs it - that way the secrets are not passed between deployments.

Reply:
removed key in outputs